package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "amat-react",
-  "author": "Geoff Hankerson ghankerson@mpr.org",
-  "version": "0.1.16",
+  "name": "@apmg/amat",
```

Comment:
We have a Ruby library called amat  ...

Reply:
how about `@apmg/amat-js`?

Reply:
sounds great

Reply:
I guess it could be `@apmg/amat-react`, but I think `@apmg/amat` should be fine because the ruby library will never be distributed via NPM, and the fact that it uses JS is implied by the very fact that it's on NPM. Can you think of a place in an app where it would become unclear which you were using?